root.go

```diff
@@ -25,7 +25,9 @@ var RootCmd = &cobra.Command{
 	Short: "Serve a RESTful API from any PostgreSQL database",
 	Long:  `Serve a RESTful API from any PostgreSQL database, start HTTP server`,
 	Run: func(cmd *cobra.Command, args []string) {
-		postgres.Load()
+		if config.PrestConf.Adapter == nil {
+			postgres.Load()
```

Comment:
for me if nil should fail with a cool message not load postgres adapter, because it can be mysql adapter for example

Reply:
if we decide to load postgres if nil we need a log message to communicate this behavior